app/src/main/res/values/strings.xml

```diff
@@ -439,8 +438,7 @@
     <string name="reading_lists_confirm_remote_delete_yes">Yes</string>
     <string name="reading_lists_confirm_remote_delete_no">No</string>
     <string name="reading_list_article_save_in_progress">The article is being downloaded, and it will be available offline when complete.</string>
-    <string name="reading_list_already_contains_selection">The chosen list already contains all of the selected pages.</string>
-    <string name="reading_list_added_articles_list_titled">Added %1$d articles to %2$s</string>
+    <string name="reading_list_articles_added_to_named">Successfully Added %1$d articles to %2$s</string>
```

Comment:
Successfully "A"dded...

The "Added" should not be capitalized for the first character.

Reply:
I think the copy will have more changes based on Robin's comment... so will do it all together...